nameko/amqp/publish.py

```diff
@@ -108,9 +108,11 @@ class Publisher(object):
     def __init__(
         self, amqp_uri, use_confirms=None, serializer=None, compression=None,
         delivery_mode=None, mandatory=None, priority=None, expiration=None,
-        declare=None, retry=None, retry_policy=None, **publish_kwargs
+        declare=None, retry=None, retry_policy=None, ssl_params=None,
```

Comment:
A little bit confused from the var naming - `ssl` above, `ssl_params` here and `ssl_options` below in tests - aren't these three the same thing? .)